### ISSUE TITLE
Parse Plist with String

### ISSUE THREAD

**Issue by jcampbell05** [CocoaPods/Xcodeproj#327]
IS it possible to expose the ability to parse a plist from a string ?

**Comment by segiddins** [comment]
Yeah, should be possible just using CF. a PR making that functionality public and testing it would be awesome!
